skimage/measure/_regionprops.py

```diff
@@ -303,6 +304,7 @@ def __init__(self, slice, label, label_image, intensity_image,
             multichannel = False
 
         self.label = label
+        self.offset = offset or np.zeros((label_image.ndim,), dtype=int)
```

Comment:
Think this might be causing the test failures

```suggestion
        self.offset = (offset is None) or np.zeros((label_image.ndim,), dtype=int)
```

Reply:
Thanks, @jakirkham!